src/framework/theme/350-400-properties-changes.md

```diff
@@ -0,0 +1,77 @@
+Nebular 4.0 unifies a lot of component properties making them more intuitive and easier to use.
+This table contains these changes per component:
+
+
+| Component     | Property                                                         | Change                                                                                                                                                                                                                                                  | Replacement                                                                                                                                                                                                                                                                                   |   |
```

Comment:
`Change` column is broken when viewing on GitHub. 
<img width="890" alt="Screen Shot 2019-05-29 at 7 12 40 PM" src="https://user-images.githubusercontent.com/6151971/58573223-f964d500-8245-11e9-84cf-e7c61b29b191.png">